native-schema-registry/python/PyGsrSerDe/AvroSerDe.py

```diff
@@ -0,0 +1,74 @@
+import fastavro
+from io import BytesIO
+
+
+class AvroSerializer:
```

Comment:
Should we define an abstract base class for data format serializers and have AvroSerializer inherit it?